app/spdb/build.sbt

```diff
@@ -250,12 +250,12 @@ def dnavarro(version: Version) = AppConfig(
     "edu.gemini.auxfile.fits.dest"               -> "/gemsoft/var/data/ictd/test/GS@SEMESTER@/@PROG_ID@",
     "edu.gemini.auxfile.fits.host"               -> "gsconfig.gemini.edu",
     "edu.gemini.auxfile.other.dest"              -> "/gemsoft/var/data/finder/GSqueue/Finders-Test/@SEMESTER@/@PROG_ID@",
-    "edu.gemini.auxfile.root"                    -> "/Users/jnavarro/.auxfile",
+    "edu.gemini.auxfile.root"                    -> "/Users/dnavarro/.auxfile",
```

Comment:
Oh, now I understand why I had to create the `.auxfile` dir manually. My global username is *jnavarro* but in svn, where the credentials are taken from, it was created as *dnavarro*. I don't know if that has changed recently.

@olesjasmirnova it's better if you get an account on your own because if I had to setup `ocs` again I may run into trouble without knowing why.